packages/sources/vue/src/components/overlays/VtmnSnackbar/VtmnSnackbar.vue

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+import '@vtmn/css-snackbar/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnSnackbar',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    content: {
+      type: String as PropType<String>,
+      required: true,
+    },
+    withCloseButton: {
+      type: Boolean as PropType<Boolean>,
+      default: false,
+    },
+    actionLabel: {
+      type: String as PropType<String>,
+      default: undefined,
+    },
+    closeCallBack: {
+      type: Function as PropType<Function>,
+      required: true,
+    },
+    actionCallBack: {
+      type: Function as PropType<Function>,
+      required: true,
+    },
```

Comment:
Can you please use "onClose" and "onAction" as we did with `VtmnSnackbar` svelte component?